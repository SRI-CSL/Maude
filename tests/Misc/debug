#!/bin/sh

MAUDE_LIB=$srcdir/../../src/Mixfix
export MAUDE_LIB

../../src/Mixfix/maude \
  < $srcdir/debug.maude -no-banner \
  > debug.out 2>&1

diff $srcdir/debug.expected debug.out > /dev/null 2>&1
