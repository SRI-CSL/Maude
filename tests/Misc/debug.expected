Maude> ==========================================
reduce in BAR : a .
==========================================
reduce in BAR : a .
rewrites: 2
result Bar: c
rewrites: 2
result Bar: c
==========================================
reduce in META-LEVEL : metaReduce(['BAR], 'a.Bar) .
*********** equation
eq [Q:Qid] = mod Q:Qid is
  protecting Q:Qid .
  sorts none .
  none
  none
  none
  none
  none
endm .
Q:Qid --> 'BAR
['BAR]
--->
mod 'BAR is
  protecting 'BAR .
  sorts none .
  none
  none
  none
  none
  none
endm
rewrites: 4
result ResultPair: {'c.Bar,'Bar}
Maude> Bye.
